### PR TITLE
Add info from Report Tool

### LINKED ISSUE #849: More Info - Shoe Story
End user submitted issue from page: [/years/04_05/shoe_story/](https://history.newtheatre.org.uk/years/04_05/shoe_story/)
---
https://www.oliverjameshymans.com/theshoestory
*nathan*

### PR DESCRIPTION
**Pull request by BenWoodford76**
Adding info from https://www.oliverjameshymans.com/

closes #849

### CODE REVIEW

File: _shows/04_05/shoe_story.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+venue: Sweet Venue
 
 crew:
   - role: Producer
     name: Laura Dexter
 ---
+
+The Shoe Story is an original and contemporary piece of theatre for families and young audiences structured as a triptych of cautionary stories all with one thing in common: shoes! The three tales involve a naughty boy who can’t tie his own shoe laces and ends up in a big knot; an evil queen with the world’s largest collection of shoes who refuses to share her wealth; and finally the little birdie-boy, born with the feet of a bird who struggles to find a pair of shoes to fit him.
+
+---
```
Comment (wjdp) [inline _shows/04_05/shoe_story.md]: `---` shouldn't be here

File: _shows/04_05/the_radiator.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+playwright: L. S. Uglow
+student_written: false
+period: Autumn
+season: In House
+season_sort: 70
+venue: New Theatre
+
+crew:
+  - role: Set Designer
+    name: Oliver James Hyman
```
Comment (wjdp) [inline _shows/04_05/the_radiator.md]: Name should end with s?

**Comment by wjdp** [comment]
For the radiator can we ad his webpage copy as `trivia`? 

https://www.oliverjameshymans.com/the-radiator

**Comment by wjdp** [comment]
Hilarious, the netlify build box ran out of memory.

**Comment by nhyty** [comment]
For the trivia bit do you want m to take the quote or just add the link?

**Comment by wjdp** [comment]
@nhyty Add the quote, there's no link support with trivia

**Comment by wjdp** [comment]
@nhyty Can you check over and confirm there's nothing else to add?

**Comment by nhyty** [comment]
@wjdp There are the reviews but I can create another issue for that.

**Pull request merged**